internal/pendingpool/pendingpool.go

```diff
@@ -0,0 +1,85 @@
+package pendingpool
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"errors"
+
+	"github.com/SIGBlockchain/project_aurum/internal/constants"
+	"github.com/SIGBlockchain/project_aurum/internal/producer/src/contracts"
+	"github.com/SIGBlockchain/project_aurum/internal/producer/src/hashing"
+	"github.com/SIGBlockchain/project_aurum/internal/producer/src/validation"
+	"github.com/SIGBlockchain/project_aurum/internal/publickey"
+)
+
+//PendingData contains pending balance and pending nonce
+type PendingData struct {
+	PendingBal   uint64
+	PendingNonce uint64
+}
+
+//PendingMap contains a map that maps a hex encoded string of a wallet address to a pointer of PendingData
+type PendingMap struct {
+	Sender map[string]*PendingData
+}
+
+//NewPendingData returns an instance of pendingData given pending balance and pending nonce
+func NewPendingData(pendingBal uint64, pendingNonce uint64) PendingData {
+	return PendingData{pendingBal, pendingNonce}
+}
+
+//NewPendingMap returns an instance of pendingMap given a wallet address and an instance of pendingData
+func NewPendingMap(walletAddress []byte, data PendingData) PendingMap {
+	encodedWalletAddr := hex.EncodeToString(walletAddress)
```

Comment:
Can this map be instantiated without passing in anything? Or I guess I don't see why it needs to be instantiated with at least one map element.